stl/inc/random

```diff
@@ -4218,10 +4218,8 @@ public:
         }
 
         void _Init(_Ty _Kx0, double _Px0) { // initialize
-            _STL_ASSERT(0.0 < _Kx0, "invalid max argument for "
-                                    "negative_binomial_distribution");
-            _STL_ASSERT(0.0 < _Px0 && _Px0 <= 1.0, "invalid probability argument for "
-                                                   "negative_binomial_distribution");
+            _STL_ASSERT(0.0 < _Kx0, "invalid max argument for negative_binomial_distribution");

```

Comment:
I don't understand what changed here such that we can now fit this string on one line... shouldn't clang-format do a consistent thing here?

Reply:
clang-format will split-and-wrap overlong comments and string literals, but it will not merge comments and string literals on consecutive lines even if they wouldn't produce an overlong line. The idea begin that if you introduce line breaks in comments and string literals you do so to achieve some specific format that the tool shouldn't upset.

TLDR: Nothing changed here to allow the merge.

Reply:
Thanks for the explanation :)